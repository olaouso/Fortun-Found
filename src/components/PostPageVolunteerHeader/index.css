.postHeaderText p {
  display: flex;
  align-items: center;
  font-family: "Acme";
  color: #590995;
  font-size: 3em;
  text-align: center;
  padding: 5%;
}
.postHeader {
  margin-bottom: 2%;
  margin-top: 2%;
}
