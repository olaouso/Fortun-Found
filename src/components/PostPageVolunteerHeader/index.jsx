import React from "react";
import { Row, Col, Container } from "react-bootstrap";
import { useTranslation } from "react-i18next";
import headerPic from "./image/headerPic.jpg";
import "./index.css";

export default function PostPageVolunteerHeader() {
  const { t } = useTranslation();

  return (
    <Container fluid="md" className="postHeader">
      <Row>
        <Col md={6}>
          <img
            className="headerPicture"
            src={headerPic}
            alt={t("postPage.altVolunteerHeader")}
          />
        </Col>
        <Col className="postHeaderText">
          <p>{t("postPage.title")}</p>
        </Col>
      </Row>
    </Container>
  );
}
