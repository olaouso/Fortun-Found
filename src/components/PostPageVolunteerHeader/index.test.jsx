import React from "react";
import renderer from "react-test-renderer";
import PostPageVolunteerHeader from "./index";

it("renders PostPageVolunteerHeader component correctly", () => {
  const tree = renderer.create(<PostPageVolunteerHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
