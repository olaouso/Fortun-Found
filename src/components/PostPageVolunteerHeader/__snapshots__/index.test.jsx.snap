// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PostPageVolunteerHeader component correctly 1`] = `
<div
  className="postHeader container-md"
>
  <div
    className="row"
  >
    <div
      className="col-md-6"
    >
      <img
        alt="postPage.altVolunteerHeader"
        className="headerPicture"
        src="headerPic.jpg"
      />
    </div>
    <div
      className="postHeaderText col"
    >
      <p>
        postPage.title
      </p>
    </div>
  </div>
</div>
`;
